tools/runqslower_example.txt

```diff
@@ -5,8 +5,15 @@ runqslower shows high latency scheduling times between tasks being
 ready to run and them running on CPU after that. For example:
 
 # runqslower
-Tracing run queue latency higher than 10000 us
-TIME     COMM             PID           LAT(us)
+
+Note: Showing TID (thread id) in the report column. The smallest
+execution unit becomes a TID when using the --pid flag as
+in that case the tool reports not only the parent pid but
+its children threads as well.
```

Comment:
Please do make sure when "-p" is specified, the above contents are presented.

Reply:
Do you mean to emit (the below caveat for example) when running the tool with `--pid` the text:

```
Note: when using the --pid flag the tool reports both the parent pid and any possibile children threads as well.
```